packages/devtools_app/lib/src/memory/flutter/memory_graph_model.dart

```diff
@@ -0,0 +1,439 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:vm_service/vm_service.dart';
+
+import '../flutter/memory_controller.dart';
+
+// TODO(terry): Ask Ben, what is a class name of ::?
+/// Internal class names :: automatically filter out.
+const String internalClassName = '::';
+
+// TODO(terry): Bake in types instead of comparing a fully qualified class name.
+const String predefinedNull = 'dart:core,Null';
```

Comment:
nit: make these be simple const objects with library and class fields  instead of comma separated strings

Reply:
Done.  Bit of work needed operator == overloading but its much nicer.